client/a8c-for-agencies/sections/marketplace/products-overview-v2/style.scss

```diff
@@ -62,42 +52,26 @@ $action-panel-height-mobile: 128px;
 	}
 
 	.product-category-menu {
+		transition: height .25s ease-out;
 		opacity: 1;
-		transition: opacity .25s ease-out;
+		height: 156px;
 	}
 
-	.product-action-panel {
+	.products-overview-v2__action-panel-wrapper {
 		position: sticky;
-		top: $header-compact-height-mobile;
 		z-index: 2;
-
-		@include break-medium {
-			top: $header-compact-height;
-		}
-	}
-
-	.product-listing-section__header-wrapper {
-		position: sticky;
-		padding-block: 16px;
-
-		top: calc( $header-compact-height-mobile + $action-panel-height-mobile );
-
-		@include break-medium {
-			top: calc( $header-compact-height + $action-panel-height );
-		}
```

Comment:
We will implement this calculation in React rather than CSS, as we are addressing dynamic header height.